python-main.js

```diff
@@ -1331,16 +1335,24 @@ function web_editor(config) {
         $("#command-share").click(function () {
             doShare();
         });
-        $("#command-connect").click(function () {
-            if ($("#command-connect").length) {
-                doConnect();
-            } else {
-                doDisconnect();
+        if (navigator.usb) {
+            $("#command-connect").click(function () {
+                if ($("#command-connect").length) {
+                    doConnect();
+                } else {
+                    doDisconnect();
+                }
+            }); 
+            $("#command-serial").click(function () {
+                doSerial();
+            });  
+        }else{
+            function buttonClicked(){
+                openModalMsg("WebUSB Error", "With WebUSB you can program your micro:bit and connect to the serial console directly from the online editor.<br/>However this feature is not supported on this browser.<br/>Click Ok to find out more. <a href=help.html#WebUSB>Ok</a>");
```

Comment:
`WebUSB Error` might sound scary for novice users. Can we replace the title with `WebUSB not supported`.